scrapy/utils/curl.py

```diff
@@ -33,6 +33,44 @@ def error(self, message):
 for argument in safe_to_ignore_arguments:
     curl_parser.add_argument(*argument, action='store_true')
 
+def extract_headers_cookies(parsed_args):
```

Comment:
Please, use `_` as a prefix for functions like these that are not meant to be imported by users.